test_cuda.sh

```diff
@@ -3,12 +3,31 @@
 
 current_dir=`pwd`
 script_dir=`dirname $0`
+CUDA_VERSION=`nvcc --version | grep release | awk '{ print $(NF-1) }' | sed "s/,//g"`
+if [[ $CUDA_VERSION != "" ]]; then
+	printf "Using Cuda %s\n" $CUDA_VERSION >&2
+else
+	printf "Error: nvcc command is not working.\n" >&2
+	exit 1
+fi
+if [[ "$4" != "" ]]; then
+	for T in $4; do
+		TARGET_SUPPORTED=`nvcc --list-gpu-arch | grep $T`
```

Comment:
`nvcc --list-gpu-arch` option is in cuda v11.1.1, but not cuda v11.0.3 or earlier. I was able to compile with cuda v11.1.1 11.2.2 11.5.2 11.6.1 11.8.0 ^^ I will test on a different hardware later today, and will report if I found any issue

Reply:
Thank you! New commit should solve that :-)